src/annotator/config.js

```diff
@@ -10,15 +10,15 @@ var docs = 'https://h.readthedocs.io/en/latest/embedding.html';
  *
  * @param {Window} window_ - The Window object to read config from.
  */
-function config(window_) {
-  var options = {
+function configure(window_) {
```

Comment:
I think this name is a mistake. `configure` is a verb, which implies that it modifies something or _applies_ the configuration. What is actually happening here is that the configuration is being _read_ from the page. `readConfig` would be fine.